packages/components/psammead-navigation/src/index.jsx

```diff
@@ -109,43 +118,91 @@ const CurrentLink = ({ children: link, script, currentPageText }) => (
   </>
 );
 
-export const NavigationUl = ({ children, ...props }) => (
-  <StyledUnorderedList role="list" {...props}>
-    {children}
+CurrentLink.propTypes = {
+  children: string.isRequired,
+  script: shape(scriptPropType).isRequired,
+  currentPageText: string,
+};
+
+CurrentLink.defaultProps = {
+  currentPageText: null,
+};
+
+export const NavigationUl = ({ children, isSwipeable, ...props }) => (
+  <StyledUnorderedList role="list" isSwipeable={isSwipeable} {...props}>
+    {React.Children.map(children, child =>
+      React.cloneElement(child, { isSwipeable }),
+    )}
   </StyledUnorderedList>
 );
 
+NavigationUl.propTypes = {
+  children: node.isRequired,
+  isSwipeable: bool,
+};
+
+NavigationUl.defaultProps = { isSwipeable: false };
+
 export const NavigationLi = ({
   children: link,
   url,
   script,
   currentPageText,
   active,
   service,
+  isSwipeable,
   ...props
-}) => (
-  <StyledListItem role="listitem">
-    {active && currentPageText ? (
-      <StyledLink
-        href={url}
-        script={script}
-        currentLink="true"
-        service={service}
-        {...props}
-      >
-        <CurrentLink script={script} currentPageText={currentPageText}>
+}) => {
+  const tabIndex = isSwipeable && { tabIndex: -1 };
+
+  return (
+    <StyledListItem role="listitem">
+      {active && currentPageText ? (
+        <StyledLink
+          href={url}
+          script={script}
+          currentLink="true"
```

Comment:
Looking good! I realise this hasn't been changed in this PR, but why is `currentLink` passed as a string instead of a boolean?

Reply:
Done